repository/n.json

```
@@ -751,6 +751,18 @@
 				}
 			]
 		},
+		{
+			"name": "Nilesoft Shell Script",
+			"details": "https://github.com/abdalmoniem/Nilesoft-Shell-Script",
+			"labels": ["syntax highlighting", "nilesoft shell", "nss"],
```

```suggestion
			"labels": ["language syntax", "nilesoft shell", "nss"],
```

"language syntax" is the canonical label for a package like this.